example/build.gradle

```diff
@@ -30,11 +30,6 @@ android {
     }
 
     buildTypes {
-        debug {
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
```

Comment:
Why was this removed? This library relies on reflection (and there's an open issue to introduce a variant of the example app that relies on said reflection), so I'd like this to remain.